### DISCUSSION TITLE
23.11.02 데일리 스크럼

### DISCUSSION THREAD

**Discussion by Jeonwoojae** [petit-a-petit/MIML#53] [Daily Scrum]
.

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - 비동기 테스트 문제 해결
    - 테스트에서만 비동기 로직을 동기 로직으로 변경
  - 비동기 통합 테스트 문제 임시 해결
    - 테스트 마다 context 초기화
    - ~~deleteAll()~~
    - ~~@Transacional~~
  - 약간의 리팩토링
- 내일은 어떤 일을 하나요?
  - 발표 준비

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 내가 친구 요청 보낸 회원 조회 기능
    - 나에게 친구 요청 보낸 회원 조회 기능
    - 나와 친구인 회원 조회 기능 (개발 중)
    - 포스트맨 에러 트러블 슈팅 (아직 미해결..)
- 내일은 어떤 일을 하나요?
    - 나와 친구인 회원 조회 기능
    - 발표 준비
